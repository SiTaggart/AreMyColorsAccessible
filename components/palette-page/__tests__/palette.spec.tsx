/* eslint-env jest */

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import PalettePage from '../';

describe('Palette Page', (): void => {
  it('renders without crashing', (): void => {
    ReactDOM.render(<PalettePage />, document.createElement('div'));
    const paletteCmp = renderer.create(<PalettePage />).toJSON();
    expect(paletteCmp).toMatchSnapshot();
  });
});
