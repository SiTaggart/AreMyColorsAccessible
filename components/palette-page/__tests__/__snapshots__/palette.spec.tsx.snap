// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Palette Page renders without crashing 1`] = `
Array [
  .emotion-6 {
  text-align: center;
}

.emotion-2 {
  font-weight: 700;
  line-height: 1;
  margin: 3rem 0;
}

.emotion-0 {
  display: inline-block;
  font-size: 2.5rem;
  margin-bottom: 0.5rem;
  width: 100%;
}

.emotion-4 {
  background: rgba(255,255,255,0.1);
  border: solid 1px;
  border-radius: 3px;
  font-size: 1.2rem;
  padding: 0.5rem 1rem;
  margin-bottom: 0;
  width: 100%;
}

<form
    className="emotion-6 emotion-7"
    onSubmit={[Function]}
  >
    <div>
      <h1
        className="emotion-2 emotion-3"
      >
        <label
          className="emotion-0 emotion-1"
          htmlFor="palette-form-input"
        >
          Add the colours from your palette
        </label>
      </h1>
      <input
        autoComplete="off"
        className="emotion-4 emotion-5"
        id="palette-form-input"
        name="paletteFormInput"
        placeholder="e.g. #ccc, #fff or blue"
        type="text"
      />
    </div>
  </form>,
  .emotion-4 {
  margin-top: 3rem;
  overflow-x: auto;
}

.emotion-2 {
  border-collapse: collapse;
  width: 100%;
}

<section
    className="emotion-4 emotion-5"
  >
    <table
      className="emotion-2 emotion-3"
    >
      <thead>
        <tr
          className="emotion-0 emotion-1"
          data-test="colorMatrix-tr"
        >
          <td
            style={
              Object {
                "width": "6rem",
              }
            }
          />
        </tr>
      </thead>
      <tbody />
    </table>
  </section>,
]
`;
